bin/dev/install_tools.sh

```diff
@@ -1,90 +1,54 @@
 #!/bin/bash
 set -o errexit -o nounset
 
-# Get version information
-. "$(dirname "$0")/versions.sh"
+# Get version information and set destination paths
+. "$(dirname "$0")/../common/versions.sh"
+
+if id -u vagrant >& /dev/null; then
+  SCF_BIN_DIR="${SCF_BIN_DIR:-/usr/local/bin}"
```

Comment:
Yes, currently in https://github.com/SUSE/cf-ci/blob/harts-provision-kvm-k8s-host/k8s-kvm/provision_k8s_host_kvm.sh, though it sounds like this will be moving to a Triton-based deploy?